client/src/components/Profile/style.css

```diff
@@ -17,4 +22,12 @@
 .bioText {
     padding: 0;
     margin-top: -2;
-}
\ No newline at end of file
+}
+
+.container{
```

Comment:
Even though you didn't do anything to .container here we should delete it because there's already a .container class in the main index.css file

Reply:
done